.gitignore

```diff
@@ -0,0 +1,11 @@
+# Compiled python modules.
+*.pyc
```

Comment:
will update. I think a couple of times I mixed up `python3` with `python` and `pip`/`pip3` on my dev machine, so likely just a stupid mistake 😅  but fixes incoming.